pages/Page.vue

```diff
@@ -1,7 +1,158 @@
+	link(href='https://fonts.googleapis.com/css2?family=Outfit:wght@600&display=swap' rel='stylesheet')
+html
+	head
+		
+		link(rel='stylesheet' href='/stylesheets/main.css')
+	.bg-blue.w-screen(class="h-[80px]")
+	img.bg-orange-300.-mt-16.mx-auto(class="w-[122px] h-[122px] rounded-[8px]" src=profile)
+	.text-gray-dark.font-poppins.text-2xl.text-center.font-semibold(style="text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);") {{data.page_name}}
+	.row.text-center
+		.text-gray-dark.font-poppins.text-md.inline-block.text-center {{data.day_of_birth}}
+		.text-gray-dark.font-poppins.text-md.inline-block.whitespace-pre  - 
+		.text-gray-dark.font-poppins.text-md.inline-block {{data.day_of_passing}}
+	.row.p-3
+		NuxtLink.mx-16.p-3.px-6.pt-2.text-white.bg-orange-400.font-poppins(style="font-weight: 700; border-radius: 32px;" to='/pages') Back
+	.container.bg-gray-light.mx-auto.h-auto.font-poppins(class="w-11/12 rounded-[8px]" style="box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);")
+		//ul.inline-flex.py-1.px-1.w-full(id="tabs")
+			li.bg-white.text-xs.text-center.px-auto.py-2(class="rounded-[8px] w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" id="default-tab" href="#obituary") Obituary
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#services") Services
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#donations") Donations
+			li.text-xs.text-center.px-auto.py-2(class="w-1/4")
+				a.font-poppins.text-center.font-semibold(style="color:#383838; text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.25);" href="#media") Media
+
+		.div(style="background: #F8F8F8;" id="tab-contents")
+			.div.px-8.py-4(style="color: #6E6E6E; font-weight: 500; font-size: 14px; line-height: 28px; letter-spacing: -0.078px; word-break: break-word;" id="obituary") {{data.obituary}} Obituary 
+			.div.p-4(id="services")
+				if visitation_date
+					.px-5.py-4.text-gray-dark.font-poppins.text-2xl.text-left.font-bold(style="line-height: 36px; text-shadow: 3px 3px 4px rgba(0, 0, 0, 0.25);") Visitation
+					.px-5.pt-2.pb-8.font-outfit.text-dark-blue(style="font-size: 20px; line-height: 30px;") {{data.visation_description}}
+					.grid.gap-x-3.gap-y-0.grid-cols-5.grid-rows-2
+						img.px-4(src="/images/clock_icon.png")
+						.py-1.font-outfit.text-dark-blue.col-span-4(style="font-size: 20px; line-height: 30px;") {{data.visitation_date + ", " + data.timezone}}
+						img.px-2(src="/images/location_icon.png")
+						.py-2.font-outfit.text-dark-blue.col-span-4.whitespace-normal(style="font-size: 20px; line-height: 30px;" id="visitation_location") visitation location
+				if funeral_date
```

Comment:
Update template usage - this needs to use Vue's interpolation and conditional rendering

Reply:
so use v-if instead of if here?